### ISSUE TITLE
HTML-Proofer ignores 'data-proofer-ignore'

### ISSUE THREAD

**Issue by r-brown** [gjtorikian/html-proofer#306]
Hello,

I'm trying to add _'data-proofer-ignore'_ attribute to the code below (Jekyll powered site):
https://github.com/Labs64/netlicensing.io/blob/gh-pages/_includes/google-search.html#L13

... but for some reason, this advice is being ignored by HTML-Proofer and I'm getting following:
```
- ./_site/blog/index.html
  *  Namespace prefix gcse is not defined (line 1)
  *  Tag gcse:search invalid (line 1)
htmlproof 2.6.4 | Error:  HTML-Proofer found 2 failures!
```

Those are used CLI attributes:
`bundle exec htmlproof ./_site --verbose --check-html --ignore-script-embeds --href-ignore '/.*www.linkedin.com.*/,#'`

What I'm missing here?

Best, Ready

**Comment by gjtorikian** [comment]
Thanks very much for the detailed report.

I've confirmed that this is a problem locally. I'll work on a fix for the next version, due out soon.

**Comment by r-brown** [comment]
Thank you for picking up this issue.
May be you could extend the format, so also other attribute declarations will be taken into account
e.g.
```
<gcse:search data-proofer-ignore></gcse:search>
<gcse:search data-proofer-ignore=""></gcse:search>
<gcse:search data-proofer-ignore="true"></gcse:search>
```

**Comment by gjtorikian** [comment]
Closed by HTMLProofer 3.0: https://github.com/gjtorikian/html-proofer/releases/tag/v3.0.0
